crates/proto/src/serialize/txt/trust_anchor.rs

```diff
@@ -0,0 +1,285 @@
+//! Trust anchor file parser
+//!
+//! A trust anchor file largely adheres to the syntax of a zone file but may only contain
+//! DNSKEY records
+
+use std::{borrow::Cow, str::FromStr as _};
+
+use crate::{
+    rr::{dnssec::rdata::DNSKEY, DNSClass, Name, Record, RecordType},
+    serialize::txt::ParseErrorKind,
+};
+
```

Comment:
(Tiny nit: let's get rid of this empty line and generally keep `crate` and `super` imports together?)